modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -61,6 +61,9 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`disableRedirect`
+| `disableRedirect` controls whether to route all data through the registry, rather than redirecting to the backend. Defaults to false.
```

Comment:
* Should it just start with "Controls" to be more consistent with how the rest of the descriptions start?
* Should "backend" be "back end" (per ISG, as a noun)?
* Should "false" be in backticks?

Reply:
Thanks! Updated.